docs/src/components/Card.js

```diff
@@ -15,11 +16,12 @@ export default function Card({
   children,
   description,
   heading = true,
+  id,
   name,
   stacked = false,
 }: Props) {
   return (
-    <Box>
+    <Box {...(id ? { id } : {})}>
```

Comment:
`<Box id={id}` >` should fine since if its undefined itll just skip it in the DOM?